youtube-dl-server.py

```diff
@@ -43,7 +43,8 @@ def q_size():
 def q_put():
     url = request.forms.get("url")
     options = {
-        'format': request.forms.get("format")
+        'format': request.forms.get("format"),
+        'proxy': request.forms.get("proxy")
```

Comment:
Do we really need the proxy to be configurable on each request? Would setting it via an env var be enough?

Reply:
Well at least for me a static setting is not enough, since I use this sometimes to avoid country restrictions or go via Tor, which is not working for every platform. I do not know if anyone else wants the same, but I thought I could share it if so.